### DISCUSSION TITLE
Power off generator

### DISCUSSION THREAD

**Discussion by gerolymatos** [jgyates/genmon#1208] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).
- [X] I have included the generator make and model in my submission.

### Version

Version V1.19.04
Generac 22K

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have not submitted logs

### What is/are your question(s)?

When a power outage occurs, Is there a way to manually turn off the generator via genmon. There have been times when we didnt want the generator to run for a few hours during the night. If not, is there other ways to turn it off without having to go outside to the generator?

**Comment by skipfire** [comment, marked as answer]
The controller does not allow that, it ignores stop commands when it is running in a power outage.  2-wire with an electronically controlled I believe would allow it, but I don't know what the conversion is or what drawbacks 2-wire may have.

**Reply by jgyates** [reply]
this video shows how to do this. 

https://www.youtube.com/watch?v=ssZ1m-kyEek

The genmon software will not allow this to occur automatically due to the firmware of the controller's remote commands only allow a stop if a remote start is issued first.

**Reply by gerolymatos** [reply]
Thank you for the reply!
